doc/ref_model/chapters/chapter07.md

```diff
@@ -311,40 +348,45 @@ The static file system includes a set of important files and folders which do no
 This allows the attestation server to detect any tampering with the static file system during the runtime of the platform.
 
 <a name="7.7"></a>
-## 7.7 Common security standards
+## 7.7 Open Source Software Security 
 
-The Cloud Infrastructure Reference Model and the supporting architectures are not only required to optimally support networking functions, but they must be designed with common security principles and standards from inception.  These best practices must be applied at all layers of the infrastructure stack and across all points of interconnections with outside networks, APIs and contact points with the NFV network functions overlaying or interacting with that infrastructure.
-Standards organizations with recommendations and best practices, and certifications that need to be taken into consideration include the following examples. However this is by no means an exhaustive list, just some of the more important standards in current use.
+Software supply chain safety is crucial and can be a complex task in virtualised and containerized environments. Open source code is present in Cloud Infrastructure software from host Operating System to virtualisation layer components, the most obvious being represented by Linux, KVM, QEMU, OpenStack, and Kubernetes. Workloads components can also be composed of open source code. The proportion of open source code to application source code is variable. It can be partial or total, visible or not. Open source code can be upstream code coming directly from open source public repositories or code within a commercial application or network function. To ensure the security of the whole system, all software and hardware components must reach the same level of security by following best security practices including secure lifecycle management. The SAFECode paper “Managing Security Risks Inherent in the Use of Third-party Components” provides a detailed risk management approach.
```

Comment:
Can we change "The proportion of open source code to application source code is variable." to "The proportion of open source code to an applications source code can vary."

Reply:
@pgoyal01 Yes, done